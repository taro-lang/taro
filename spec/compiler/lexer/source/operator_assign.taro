val number : Int32 = 1
val word : String = "hello"
var mutate : String = "mutatable"
mutate = "something new"