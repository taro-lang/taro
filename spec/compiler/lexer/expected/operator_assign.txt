Assign (=) line: 0, length: 1, position: (0,0)